### PR TITLE
Fixed DailyPlan and TimeCard confrimation

### PR DESCRIPTION
**Pull request by InEightDevOps**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by InEightDevOps** [comment]
Updated schema to fix couple of issues.

File: certified-connectors/InEight/apiDefinition.swagger.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "version": "1.0",
     "title": "InEight",
-    "description": "The InEight Connector provides a mechanism to create new data in InEight or extract data from InEight. It can be used to integrate other systems with InEight.",
+    "description": "InEight Connector provides an API to work with InEight objects.",
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiDefinition.swagger.json:6]: @InEightDevOps Please revert the description value back as the existing one looks good than the new one.

File: certified-connectors/InEight/readme.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
 # InEight Connector
-The InEight Connector provides a mechanism to create new data in InEight or extract data from InEight. It can be used to integrate other systems with InEight.
+InEight Connector provides an API to work with InEight objects.
```
Comment (sriyen-msft) [inline certified-connectors/InEight/readme.md:2]: @InEightDevOps Same here

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/InEight/apiDefinition.swagger.json (hunk 1 above)
Comment (InEightDevOps) [reply certified-connectors/InEight/apiDefinition.swagger.json:6]: Updated comments as requested.

File: certified-connectors/InEight/readme.md (hunk 2 above)
Comment (InEightDevOps) [reply certified-connectors/InEight/readme.md:2]: updated as requested

File: certified-connectors/InEight/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1645,47 +1654,50 @@
...
+                    "type": "string",
+                    "description": "DtModifiedDate"
+                  },
+                  "ReleasedDate": {
+                    "type": "string",
+                    "description": "ReleasedDate"
+                  },
+                  "DtReleasedDate": {
+                    "type": "string",
+                    "description": "DtReleasedDate"
```
Comment (sriyen-msft) [inline certified-connectors/InEight/apiDefinition.swagger.json:1688]: @InEightDevOps All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@InEightDevOps Please resolve the branch conflicts to get this PR approved and merged.

**Comment by sriyen-msft** [comment]
@InEightDevOps Please resolve the branch conflicts to get this PR approved and merged.

File: certified-connectors/InEight/apiDefinition.swagger.json (hunk 3 above)
Comment (InEightDevOps) [reply certified-connectors/InEight/apiDefinition.swagger.json:1688]: Updated description as per the comments

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**